db/Schemas/Listing.js

```diff
@@ -3,7 +3,7 @@ const mongoose = require('mongoose');
 
 
 const houseSchema = new mongoose.Schema({
-    Listing_id: Number,
```

Comment:
I did the same thing.

Reply:
Had to change it to a string to maintain the padding 👍